website/src/data/users.js

```diff
@@ -258,6 +258,15 @@ const users = [
     fbOpenSource: false,
     pinned: true,
   },
+  {
+    title: 'MateuszDabrowski',
+    description: 'Salesforce Marketing Cloud Programmatic Documentation',
+    preview: require('./showcase/md.png'),
```

Comment:
Don't see this file being added

Reply:
I see information about lack of privileges to add any files into the folder containing screenshots, that’s why I added it to this ticket.

Reply:
You have to fork the repository, and add the image to your fork.